_includes/components/reportingstatus/reporting-status-label.html

```diff
@@ -0,0 +1,15 @@
+{% assign status_css = indicator.reporting_status | slugify %}
```

Comment:
@EFT-Defra Here, instead of "indicator" you should use "include.indicator". It probably works either way, but it's a good practice to avoid using shared global variables when possible.